images/opensearch/2.Dockerfile

```diff
@@ -36,8 +34,6 @@ ENV TMPDIR=/tmp \
     # When Bash is invoked as non-interactive (like `bash -c command`) it sources a file that is given in `BASH_ENV`
     BASH_ENV=/home/.bashrc
 
-RUN yum -y install hostname && yum -y clean all  && rm -rf /var/cache
```

Comment:
The [entrypoint](https://github.com/uselagoon/lagoon-images/blob/main/images/opensearch/docker-entrypoint.sh) still depends on `hostname`, so I guess that also needs to be updated?

Reply:
no - that was removed and replaced by the existing var HOSTNAME

Reply:
Ah, that's what I get for trying to review in the Github UI :D